src/test/java/net/snowflake/client/jdbc/ConnectionIT.java

```diff
@@ -484,8 +485,8 @@ public void testWrongHostNameTimeout() throws InterruptedException
       if (TelemetryService.getInstance().isDeploymentEnabled())
       {
         assertThat("Telemetry event has not been reported successfully. Error: " +
-                   TelemetryService.getInstance().getLastError(),
-                   TelemetryService.getInstance().getFailureCount() == 0);
+                   TelemetryService.getInstance().getLastClientError(),
+                   TelemetryService.getInstance().getClientFailureCount() == 0);
```

Comment:
http://hamcrest.org/JavaHamcrest/tutorial
Use `equalTo` matcher

Reply:
changed to equalTo.